git.lua

```diff
@@ -112,6 +112,30 @@ local function branches(token)
     return res
 end
 
+local function alias(token)
```

Comment:
Please submit 'alias' feature as a separate pull request. Also i'm not verya familiar with this, so could you please clarify the purpose of this?